src/common/IOSUtils.ts

```diff
@@ -264,9 +263,22 @@ export class IOSUtils {
         udid: string,
         compName: string,
         projectDir: string,
+        appBundlePath: string | undefined,
         targetApp: string,
         targetAppArguments: LaunchArgument[]
     ): Promise<boolean> {
+        if (appBundlePath && appBundlePath.trim().length > 0) {
+            const installCommand = `${XCRUN_CMD} simctl install ${udid} '${appBundlePath.trim()}'`;
+
+            try {
```

Comment:
We should add some output to the user here too, e.g. "Installing bundle ${appBundlePath} to simulator." or something to that effect. This is a good action to inform the user about.